test/tester.cpp

```diff
@@ -15,123 +15,165 @@
 #include <sstream>
 #include <string>
 #include <climits>
+#include <cassert>
 
+#include "utils.h"
 #include "tester.h"
 
+
 static std::string headerText("Running tests");
-void setHeaderText(std::string text)
+void set_header_text(std::string text)
 {
 	headerText = text;
 }
 
 
-static pass_report_mode passMode{ pass_report_mode::indicate };
-void setPassReportMode(pass_report_mode mode)
+static pass_report_mode prm{ pass_report_mode::indicate };
+void set_pass_report_mode(pass_report_mode mode)
 {
-	passMode = mode;
+	prm = mode;
 }
 
 
-static unsigned short failThreshold{ 0 };
-void setFailThreshold(unsigned short value)
+static unsigned short fail_threshold{ 0 };
+void set_fail_threshold(unsigned short value)
 {
-	failThreshold = value;
+	fail_threshold = value;
 }
 
 
-void setMaxFailThreshold()
+void set_max_fail_threshold()
 {
-	failThreshold = USHRT_MAX;
+	fail_threshold = USHRT_MAX;
 }
 
 
 static std::ostream* pOut{ &std::cout };
-void setOutput(std::ostream& o)
+void set_output(std::ostream& o)
 {
 	pOut = &o;
```

Comment:
```suggestion
	p_out = &o;
```

Reply:
The convention is to use `p` prefix for pointers. Here, `p` does not represent a word. However, I recommend we do a survey of other code bases to see what people do in this case. Until then, I recommend not making this change.